### PR TITLE
production run using named queries

### PR DESCRIPTION
**Pull request by satterly**
move queries out of controller and into zuora class

### CODE REVIEW

**Comment by johnduffell** [comment]
good 👍

File: app/csv/Charge.scala [hunk 1]
```diff
@@ -68,6 +68,25 @@ Charge_day // day we're running for
      and RatePlanCharge.Name = 'Holiday Credit')
     */
 
+  val Rate_Plan_Charge__Charged_Through_Date = 0
```
Comment (johnduffell) [inline app/csv/Charge.scala:71]: did you mean to put these back in? maybe you are using them for the data lake import?

Comment (satterly) [reply app/csv/Charge.scala:71]: i had a merge conflict and assumed that they needed to be there. if you don't need them please delete them again. sorry. (this is exactly what you were talking about before :)

Comment (johnduffell) [reply app/csv/Charge.scala:71]: 😆 ok no problem

**Pull request merged**